vermouth/processors/do_mapping.py

```diff
@@ -644,8 +652,16 @@ def do_mapping(molecule, mappings, to_ff, attribute_keep=(), attribute_must=()):
                     else:
                         # No nodes hat the attribute.
                         graph_out.nodes[out_idx][attr] = None
+                if attr in attribute_stash:
+                    if vals:
+                        graph_out.nodes[out_idx]["_old_"+attr] = vals[0]
+                    else:
+                        # No nodes hat the attribute.
+                        graph_out.nodes[out_idx]["_old_"+attr] = None
```

Comment:
@pckroon I think this statement but also the one from attribute_keep are never triggered because if no nodes have the attribute the code will not go down here or am I missing anything?

Reply:
so you want to keep it for the stash function as well? I checked and no test triggers it anywhere, and I still think in that bit it would never go there. But also not 100% sure about it. I'm fine with it either way

Reply:
I'm fine with leaving it as it is in this PR now.